modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/CheckboxBase.tsx

```diff
@@ -132,9 +132,12 @@ export default function CheckboxBase({
 					...(otherProps.tip && {
 						'aria-describedby': `${otherProps.id ?? name}_fieldHelp`,
 					}),
-					...(otherProps.errorMessage && {
-						'aria-errormessage': `${otherProps.id ?? name}_fieldError`,
-					}),
+					...(otherProps.displayErrors &&
+						otherProps.errorMessage &&
+						!otherProps.valid && {
+							'aria-errormessage': `${otherProps.id ?? name}_fieldError`,
```

Comment:
Hi @igor-franca ... thanks for sending this. aria-errormessage has still recently had some bugs with different screen readers. So, for now we should just use aria-describedby. That's what we use most in portal.

Reply:
Hi @ethib137, in this case we are already planning to use describedby for another purpose. To announce the "help message"

Reply:
![image](https://github.com/user-attachments/assets/1a9c1dff-09f1-41e1-b0df-1855436c3cd3)

Should I concatenate [the error messages and help message](https://github.com/liferay-platform-experience/liferay-portal/blob/e55b61ce36ad7739384c9946d59efd24f293a491/modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/FieldFeedback.tsx#L27-L38) into a single feedback?

Reply:
I would recommend using [FieldBase](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx/#L121) or at least using the FieldFeedback component in the way it is used there.

Reply:
Got it! We are using different identifiers in the data engine feedback. One for help and another for error.
The frontend-js feedback is passing the identifier to `ClayForm.FeedbackGroup`

Reply:
Notice that FieldFeedback is also handling `aria-live="assertive` for error and warning messages. This is why it's better to use the FieldBase or FieldFeedback, it handles lots of little details we might otherwise forget about.